images/bootstrap/resources/keeper.yaml

```diff
@@ -62,3 +70,7 @@ spec:
         - name: stolon
           secret:
             secretName: stolon
+        - name: etcd-secrets
+            hostPath:
```

Comment:
You are right, default intend is 4, but we have 2 in yaml, I will add .editorconfig to prevent this issue happen again